tests/filecheck/dialects/emitc/emitc_ops.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: XDSL_GENERIC_ROUNDTRIP
+
+//===----------------------------------------------------------------------===//
+// CallOpaqueOp
+//===----------------------------------------------------------------------===//
+
+%0 = emitc.call_opaque "blah"() : () -> i64
+emitc.call_opaque "foo" (%0) {args = [
+  0 : index, dense<[0, 1]> : tensor<2xi32>, 0 : index
+]} : (i64) -> ()
+emitc.call_opaque "test" ()  : () -> ()
+
+// CHECK: builtin.module {
```

Comment:
I would still prefer not to check for the existence of `builtin.module` but won't insist.

Reply:
removed, started to work locally... must have done something wrong